templates/addons/calico-ipv6.yaml

```diff
@@ -12,9 +12,8 @@ data:
   # Configure the backend to use.
   calico_backend: "none"
 
-  # The CNI network configuration to install on each node.  The special
```

Comment:
@jsturtevant could you please take look to make sure I didn't undo any ipv6 specific changes in here? this is the original 1.16 template https://docs.projectcalico.org/v3.16/manifests/calico.yaml

I tested and everything looks good

Reply:
didn't see any changes specific to the configuration.  LGTM